package/container/framework-full/Dockerfile

```diff
@@ -3,12 +3,15 @@
 
 # on dockerhub hepcloud/decision-engine-ci
 FROM sl:7
-LABEL description="hepcloud/decision-engine"
+LABEL description="hepcloud/decision-engine-framework-full"
```

Comment:
We need also a decision-engine-full. That's the one that will be used for an all-in-one installation and test (e.g. it could be used for the integration tests that Vito is running, after loading the configuration). Not sure if this container should become that or if it should be a separate one here or in the decisionengine-modules repo. 
Maybe a separate one, because for CI (linters and unit tests) we don't need all the dependencies (all the RPMs the modules depend on, GlideinWMS is OK as git clone and not as RPM install). In that case the name of this could become to decision-engine-ci or decision-engine-framework because thin

Reply:
I'd recommend a separate container for the modules in that repo.  I'm tempted to eventually have the modules in the main repo, but that is for another time.

I'll rebase and see what things look like.

Reply:
Any thoughts?